.gitattributes

```diff
@@ -1 +1 @@
-* -crlf
+* eolf=lf
```

Comment:
I didn't find eolf from https://git-scm.com/docs/gitattributes. The closest is eol=lf. But I am not familiar with gitattributes ...